src/v/cluster/tx_gateway_frontend.cc

```diff
@@ -3221,6 +3221,10 @@ tx_gateway_frontend::get_all_transactions() {
           model::tx_manager_nt.ns, model::tx_manager_nt.tp, pa.id);
         auto ntp_res = co_await get_all_transactions_for_one_tx_partition(
           tx_manager_ntp);
+        if (
+          ntp_res.has_error() && ntp_res.error() == tx_errc::not_coordinator) {
+            continue;
+        }
```

Comment:
This silently the error, is this expected behavior. i.e. we may lay about ongoing transactions

Reply:
This is expected behavior, Kafka does the same. The API isn't "get the ongoing transactions of the specific partitions of the coordinator" but "get the ongoing transactions of all txn coordinatror's partitions hosted a broker". Client sends requests to all brokers and then aggregate the results. There is no coordination and it's ok to skip over `not_coordinator` (in happy case a leader is on another broker and will be included into the results).

https://github.com/redpanda-data/redpanda/pull/12130#discussion_r1267326889